sql/rpl_replica.cc

```diff
@@ -7065,22 +7065,10 @@ wsrep_restart_point :
   rli->events_until_exit = abort_slave_event_count;
 #endif
 
-<<<<<<< HEAD
-  thd = new THD;                     // note that contructor of THD uses DBUG_ !
-  thd->thread_stack = (char *)&thd;  // remember where our stack is
-  mysql_mutex_lock(&rli->info_thd_lock);
-  rli->info_thd = thd;
-||||||| merged common ancestors
-    thd = new THD;  // note that contructor of THD uses DBUG_ !
-    thd->thread_stack = (char *)&thd;  // remember where our stack is
-    mysql_mutex_lock(&rli->info_thd_lock);
-    rli->info_thd = thd;
-=======
     thd = new THD;  // note that constructor of THD uses DBUG_ !
```

Comment:
This part is interesting because upstream has indentation here, but PXC has a label before this block, which causes clang to think the indentation should be smaller.
Fixed.